packages/mds-stream-processor/labelers/index.ts

```diff
@@ -0,0 +1,23 @@
+import db from '@mds-core/mds-db'
+import { UUID, VEHICLE_TYPE, PROPULSION_TYPE } from '@mds-core/mds-types'
+
+type MessageLabeler<TMessage, TLabel> = (message: TMessage) => Promise<TLabel>
+
+export interface DeviceLabel {
+  vehicle_type: VEHICLE_TYPE
+  vehicle_propulsion: PROPULSION_TYPE[]
+}
+
+export const deviceLabeler: MessageLabeler<{ device_id: UUID }, DeviceLabel> = async ({ device_id }) => {
+  const { type: vehicle_type, propulsion: vehicle_propulsion } = await db.readDevice(device_id)
+  return { vehicle_type, vehicle_propulsion }
+}
+
+export interface MessageLatencyLabel {
+  message_latency_ms: number
```

Comment:
In this case it's the delta between two timestamps so it's really just a number.

Reply:
I suppose, but it's a number that represents time in MS... It'd be nice to have the type be something more than `number` because it really is a defined unit of measurement IMO, but yeah, `Timestamp` might not be proper.